examples/domain_classifier_example.py

```diff
@@ -55,7 +57,6 @@ def attach_args(
 ):
     argumentHelper = ArgumentHelper(parser)
 
-    argumentHelper.add_arg_device()
     argumentHelper.add_arg_enable_spilling()
```

Comment:
I think we should probably switch the whole thing to just just below , helps with overall alignment of NeMo-Curator. 

```python3
 args = ArgumentHelper.parse_distributed_classifier_args().parse_args()
 ```

Reply:
I want the examples to only have argparse arguments for initializing the cluster, so I did something a bit different. Let me know if you agree with the way I handled this.

Reply:
This makes sense, we can go with that. Thanks